packages/engine/src/utils/os.coffee

```diff
@@ -25,9 +25,9 @@ module.exports =
         OS=$DISTRIB_ID
         VERSION=$DISTRIB_RELEASE
     # Arch Linux
-    elif uname -r | egrep 'ARCH$' >/dev/null; then
+    elif cat /etc/os-release | egrep 'arch$' >/dev/null; then
```

Comment:
If `uname` is here, then why not using uname, this way we could also support other distrib than arch ?

Reply:
`uname -r` prints same for all distributions: `4.19.121-linuxkit`. I tested on Centos and Ubuntu. So, we can't use it to determine.